server/server.jsx

```diff
@@ -45,6 +46,38 @@ app.get("/get-initial-markers", async (req, res) => {
   }
 });
 
+app.post("/filters-params", async (req, res) => {
```

Comment:
when this get's merged, we'll need to merge with/resolve conflict with existing `get-initial-taps` and` get-marker-moving-map` endpoints!